contracts/test-utils/src/lang_utils.ts

```diff
@@ -0,0 +1,9 @@
+import * as _ from 'lodash';
+
+/**
+ * _.zip() that clips to the shortest array.
+ */
+export function shortZip<T1, T2>(a: T1[], b: T2[]): Array<[T1, T2]> {
```

Comment:
This is better than `_.zip()` because the return type isn't some weird `Array<[T1 | undefined, T2 | undefined]>`, which makes `_.zip()` pretty inconvenient to use in TS.